package.json

```diff
@@ -16,7 +16,8 @@
     "dotenv": "^16.0.3",
     "express": "^4.18.2",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
```

Comment:
I get why the trailing-comma rule is a thing now. No issue here, just pointing out that it looks funny in GitHub's PR UI.